src/components/Section.js

```diff
@@ -14,8 +14,8 @@ export default (editor, { dc, coreMjmlModel, coreMjmlView }) => {
         draggable: '[data-gjs-type=mj-body]',
         droppable: '[data-gjs-type=mj-column]',
         'style-default': {
-          'padding-top': '10px',
-          'padding-bottom': '10px',
+          // 'padding-top': '10px',
+          // 'padding-bottom': '10px',
```

Comment:
I think those changes in blocks should be placed here, no?

Reply:
For this one, it actually comes from one of my users complained that he cannot get a true WYSIWYG experience because there was always padding between sections that cannot be disabled. I think it will be better to provide a UI option to toggle it, but I haven't got time to do so.